docs/errors/details/_err_ngrok_305.md

```diff
@@ -4,6 +4,6 @@ Binding a custom subdomain on an ngrok domain requires a paid plan. Once you sig
 
 ### Possible Solutions
 
-Sign up for a paid plan at https://dashboard.ngrok.com/signup You'll need the Basic plan or better.
+[Sign up for a paid plan](https://dashboard.ngrok.com/signup). You'll need the Basic plan or better.
```

Comment:
i don't think Basic is a plan name anymore. 
```suggestion
[Sign up for a paid plan](https://dashboard.ngrok.com/signup). You'll need either a Personal plan or higher or any pay-as-you-go plan.
```

Reply:
Additionally, here we don't have a list, but I noticed in the previous error details you used a list. What is the reasoning behind this?

Reply:
Ahh, I missed it. I'll double-check to make sure all the others are lists too. I thought making these into lists, even if there's just one sentence, would help readers know where to jump to when skimming, as they likely don't want to read the full error page